phpBB/mcp.php

```diff
@@ -116,15 +116,35 @@
 	);
 
 	$allow_user = false;
+	$topic_info = phpbb_get_topic_data(array($topic_id));
```

Comment:
You're changing this from being only executed if needed in an edge case to always being executed. This will probably increase the overall server load and shouldn't be done. Instead, you should probably also pass `$quickmod` to the event and then execute this if needed

Reply:
Removed these changes and added `$quickmod` to the event instead.